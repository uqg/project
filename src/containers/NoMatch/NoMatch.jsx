import React from "react";
import styles from "./NoMatch.scss";

class NoMatch extends React.Component {
	constructor(props) {
		super(props);
	}

	render() {
		return (
			<div className={styles.error}>
				No Match
			</div>
		);
	}
}



export default NoMatch;
