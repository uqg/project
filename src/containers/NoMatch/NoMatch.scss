$color-error: #ce4233;


.error
 {
    height: 50px;
    line-height: 50px;
    text-align: center;
    position: absolute;
    top: 50%;
    left: 50%;
    transform: translate(-50%, -50%);
    font-family: helvetica, arial, sans-serif;
    font-weight: 900;
    color: $color-error;
    letter-spacing: 0.2em;
    font-size: 30px;
}
