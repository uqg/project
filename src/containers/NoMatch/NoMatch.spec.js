'use strict';

import React from 'react';
import NoMatch from './NoMatch.jsx';
import renderer from 'react-test-renderer';

jest.useFakeTimers();
Date.now = jest.fn(() => 1482363367071);

it('renders correctly', () => {
  const tree = renderer.create(<NoMatch />).toJSON();
  expect(tree).toMatchSnapshot();
});