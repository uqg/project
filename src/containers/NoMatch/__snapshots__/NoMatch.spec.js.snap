exports[`test renders correctly 1`] = `
<div
  className="error">
  No Match
</div>
`;
